src/lib/components/particles/ad-slot/stories.module.scss

```diff
@@ -0,0 +1,22 @@
+.adSlotContainer {
+  border: 1px solid #e0e0e0;
+
+  &::before {
+    content: 'Advertisement';
+  display: block;
+  position: relative;
+  font-family: GuardianTextSans, "Guardian Text Sans Web", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+  font-size: 0.75rem;
+  line-height: 1.3;
+  font-weight: 400;
+  --source-text-decoration-thickness: 2px;
+  height: 24px;
+  max-height: 24px;
+  background-color: #F6F6F6;
```

Comment:
I wonder if this should support dark mode on the apps. In that case you could replace it with `var(--tertiary-bg-color)`